WalletWasabi/WabiSabi/Client/Batching/FinishedPayment.cs

```diff
@@ -0,0 +1,9 @@
+using NBitcoin;
+
+namespace WalletWasabi.WabiSabi.Client.Batching;
+
+// Represents a payment that was made successfully in a coinjoin transaction.
```

Comment:
> Tripple-slash doc has better tooling support:

True. But I heat them because they are noise as fuck and make the code much harder to read, costly to maintain (no body updates them) while provide nothing useful.